cli/azd/pkg/azdo/utils.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azdo
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/environment"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+)
+
+// helper method to verify that a configuration exists in the .env file or in system environment variables
+func ensureConfigExists(ctx context.Context, env *environment.Environment, key string, label string) (string, error) {
```

Comment:
Given that this returns the environment variable value, should this named as `LookupEnv` instead of `ensureConfigExists`?